eng/Versions.props

```
@@ -8,7 +8,7 @@
     <MajorVersion>3</MajorVersion>
     <MinorVersion>8</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>4</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>0</PreReleaseVersionLabel>
```

I got this wrong last time
```suggestion
    <PreReleaseVersionLabel>1</PreReleaseVersionLabel>
```